apps/condo/domains/contact/components/contactRoles/ContactRoleCard.tsx

```diff
@@ -0,0 +1,85 @@
+import { InfoCircleOutlined } from '@ant-design/icons'
+import styled from '@emotion/styled'
+import { Alert, Typography } from 'antd'
+import { get } from 'lodash'
+import Link from 'next/link'
+import React, { CSSProperties, useMemo } from 'react'
+
+import { useIntl } from '@core/next/intl'
+
+import { colors, fontSizes } from '@condo/domains/common/constants/style'
+import { TicketPropertyHint, TicketPropertyHintProperty } from '@condo/domains/ticket/utils/clientSchema'
+
+import { useTicketPropertyHintContent } from '@condo/domains/ticket/hooks/useTicketPropertyHintContent'
+
+const StyledAlert = styled(Alert)`
+  background-color: ${colors.successBG};
+  border: none;
+  
+  .ant-alert-description > div {
+    overflow: hidden;
+  }
+  
+  & svg, & > .ant-alert-content > .ant-alert-message {
+   color: ${colors.black}; 
+  }
+`
+
+const TEXT_STYLES: CSSProperties = { color: colors.black }
+const LINK_STYLES: CSSProperties = { ...TEXT_STYLES, position: 'relative', bottom: '-7px', fontSize: fontSizes.content }
+
+type TicketPropertyHintCardProps = {
+    propertyId: string
+    hintContentStyle?: CSSProperties
+}
+
+export const ContactRoleCard: React.FC<TicketPropertyHintCardProps> = ({ propertyId, hintContentStyle }) => {
```

Comment:
This component is identical to `TicketPropertyHintCard` and is not used anywhere, I think we need to delete this file

Reply:
Deleted :)